### DISCUSSION TITLE
`min-*` media selector only works with  literal values

### DISCUSSION THREAD

**Discussion by tryoxiss** [tailwindlabs/tailwindcss#14303] [Ideas]
Currently, there is a `max-*`  selector which applies below the breakpoint, an a `min-*` selector which applies above it. However, the `max-*`  selector works with existing breakpoints and `min-*` only works with  `[literal values]`, which feels inconsistant (to me, at least).

**Comment by franktucker907** [comment]
Hello,
It sounds like you’re noticing an inconsistency in how the max-* and min-* selectors work with breakpoints. The max-* selector applies to existing breakpoints, while the min-* selector only works with literal values. This difference can indeed feel inconsistent.
Best Regards
franktucker907
